src/core.ts

```diff
@@ -244,10 +266,10 @@ export function sign (bytes: Bytes, privateKey: Bytes): Bytes {
  *
  * @returns `true` if all signatures are valid and match, `false` otherwise or if no signatures were provided
  */
-export function verifyTx (tx: StdTx, meta: SignMeta): boolean {
-    const signMsg = createSignMsg(tx, meta);
+export function verifyTx(tx: StdTx, meta: SignMeta): boolean {
+  const signMsg = createSignMsg(tx, meta);
 
-    return verifySignatures(signMsg, tx.signatures);
+  return verifySignatures(signMsg, tx.value.signatures);
```

Comment:
q: do we need `tx.value && tx.value.signatures` here to guard against null values?

Reply:
Not sure what the suggestion is here. Do you mean an "if" statement checking that `tx.value` is not null?

Reply:
Yes, that's correct.
```
'signatures' in tx.value && tx.value.signatures !== null
``` 
^ was added to `signTx` and was wondering if a similar check should occur here. Looking into it further, `verifySignatures` already performs the check with:
```
`if (signatures.length > 0)`
```